hil_extension/load-config.sh

```diff
@@ -0,0 +1,21 @@
+# For existing shell scripts to use the ../conf/quads.yml
```

Comment:
Actually, load-config.sh is already used by Red-Hat. It is in quads/bin. I was added by people from Red-Hat.

I made another copy in quads/lib to make my code completely independent of the rest Quads